taichi/runtime/llvm/internal_function.h

```diff
@@ -1,3 +1,12 @@
+#define _CHECK(cond, r)                        \
```

Comment:
Let's use `TI` as the prefix of all macros.

Reply:
Done